contrib/job-scripts/coreudmp-handling/pre.d/001-set-core-pattern

```diff
@@ -0,0 +1,14 @@
+#!/bin/sh -e
+
+if [ -z $AVOCADO_JOB_LOGDIR ]; then
+   exit 1
+fi
+
+COREDUMPS_DIR="$AVOCADO_JOB_LOGDIR/coredumps"
+mkdir $COREDUMPS_DIR
+
+# Save a backup of the core pattern setting
+cat /proc/sys/kernel/core_pattern > $COREDUMPS_DIR/.backup_core_pattern
+
+# Set core pattern
+echo "$COREDUMPS_DIR/core.%p" > /proc/sys/kernel/core_pattern
```

Comment:
Hello @clebergnu, I took a look at the `ulimit` problem you mentioned during meeting and it is possible to change the ulimit. The only problem in you code was you changed the parent process, which in case of `shell=True` is the `sh` process executing the script. This works and IMO it'd be really nice to include it (along with explanation and probably change the `ppid` to `bash_pid` to emphasize it's not yet the avocado pid.

```bash
ppid=$$
prlimit --core=unlimited: --pid $(cat /proc/$ppid/status | grep PPid | cut -f2)
```